pkg/shares/shares.go

```diff
@@ -70,3 +105,9 @@ func FromBytes(bytes [][]byte) (shares []Share) {
 	}
 	return shares
 }
+
+// numberOfBytesVarint calculates the number of bytes needed to write a varint of n
+func numberOfBytesVarint(n uint64) (numberOfBytes int) {
+	buf := make([]byte, binary.MaxVarintLen64)
+	return binary.PutUvarint(buf, n)
+}
```

Comment:
can we use this instead? https://github.com/celestiaorg/celestia-app/blob/8c4982c9b111377b9ab3566f3619cff23f472101/pkg/shares/utils.go#L14-L18 ?

Reply:
Great question and yes we can. I'll cut a new issue to consolidate these `DelimLen` and `numberOfBytesVarint` functions in a subsequent PR.